VideoLocker/src/main/java/org/edx/mobile/http/ApiConstants.java

```diff
@@ -4,20 +4,13 @@
 
     public static final String URL_MY_USER_INFO ="/api/mobile/v0.5/my_user_info";
 
-    public static final String URL_USER_INFO ="/api/mobile/v0.5/users/{username}/";
-
-
     public static final String URL_COURSE_ENROLLMENTS =
         "/api/mobile/v0.5/users/{username}/course_enrollments/?format=json";
 
 
-    public static final String URL_FB_FRIENDS_COURSE = "/api/mobile/v0.5/social/facebook/courses/friends";
-
     public static final String URL_FB_FRIENDS_IN_COURSE =
```

Comment:
Still appears to be used, to show your friends in a course when viewing the announcements screen (that seems strange)